<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modpwd</class>
 <widget class="QWidget" name="modpwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/sy.jpg</normaloff>:/sy.jpg</iconset>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="but_to_pwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>密码修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_search_row">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>查询用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_add_row_usr">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>增加用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_remove_row_usr">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>340</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>删除该行用户和密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_ent_mod">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确认修改</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="managepage_usr_input">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="managepage_usr_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="managepage_pwd_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>密 码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="managepage_usr_pwd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="managepage_sno_text">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>学 号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="managepage_sno_input">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="managepage_usr_pwd_confirm">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>200</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="managepage_pwd_text_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>确认密码</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
